### ISSUE TITLE
Error installing through pip on windows

### ISSUE THREAD

**Issue by klaipher** [sumerc/yappi#29]
I tried to install yappi and got this error. I use Windows 10 1909 and Python 3.7. I cloned the repository from the github and removed the slash symbol from https://github.com/sumerc/yappi/blob/master/setup.py#L82. Then I installed yappi via `python setup.py install`

```(venv) C:\Users\dimam\Desktop\Programming\exchange_bot>pip install yappi
Collecting yappi
  Using cached https://files.pythonhosted.org/packages/90/8e/ac718e8ffaffb7c92c09eb2cec33557f8f1b3ec32af5b7599a993a7700c8/yappi-1.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info\yappi.egg-info
    writing pip-egg-info\yappi.egg-info\PKG-INFO
    writing dependency_links to pip-egg-info\yappi.egg-info\dependency_links.txt
    writing entry points to pip-egg-info\yappi.egg-info\entry_points.txt
    writing top-level names to pip-egg-info\yappi.egg-info\top_level.txt
    writing manifest file 'pip-egg-info\yappi.egg-info\SOURCES.txt'
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\dimam\AppData\Local\Temp\pip-install-b6ze0_h7\yappi\setup.py", line 95, in <module>
        url=HOMEPAGE,
      File "C:\Users\dimam\Desktop\Programming\exchange_bot\venv\lib\site-packages\setuptools\__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "C:\Users\dimam\AppData\Local\Programs\Python\Python37\lib\distutils\core.py", line 148, in setup
        dist.run_commands()
      File "C:\Users\dimam\AppData\Local\Programs\Python\Python37\lib\distutils\dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "C:\Users\dimam\AppData\Local\Programs\Python\Python37\lib\distutils\dist.py", line 985, in run_command
        cmd_obj.run()
      File "C:\Users\dimam\Desktop\Programming\exchange_bot\venv\lib\site-packages\setuptools\command\egg_info.py", line 296, in run
        self.find_sources()
      File "C:\Users\dimam\Desktop\Programming\exchange_bot\venv\lib\site-packages\setuptools\command\egg_info.py", line 303, in find_sources
        mm.run()
      File "C:\Users\dimam\Desktop\Programming\exchange_bot\venv\lib\site-packages\setuptools\command\egg_info.py", line 534, in run
        self.add_defaults()
      File "C:\Users\dimam\Desktop\Programming\exchange_bot\venv\lib\site-packages\setuptools\command\egg_info.py", line 570, in add_defaults
        sdist.add_defaults(self)
      File "C:\Users\dimam\AppData\Local\Programs\Python\Python37\lib\distutils\command\sdist.py", line 226, in add_defaults
        self._add_defaults_python()
      File "C:\Users\dimam\Desktop\Programming\exchange_bot\venv\lib\site-packages\setuptools\command\sdist.py", line 127, in _add_defaults_python
        build_py = self.get_finalized_command('build_py')
      File "C:\Users\dimam\AppData\Local\Programs\Python\Python37\lib\distutils\cmd.py", line 299, in get_finalized_command
        cmd_obj.ensure_finalized()
      File "C:\Users\dimam\AppData\Local\Programs\Python\Python37\lib\distutils\cmd.py", line 107, in ensure_finalized
        self.finalize_options()
      File "C:\Users\dimam\Desktop\Programming\exchange_bot\venv\lib\site-packages\setuptools\command\build_py.py", line 34, in finalize_options
        orig.build_py.finalize_options(self)
      File "C:\Users\dimam\AppData\Local\Programs\Python\Python37\lib\distutils\command\build_py.py", line 55, in finalize_options
        self.package_dir[name] = convert_path(path)
      File "C:\Users\dimam\AppData\Local\Programs\Python\Python37\lib\distutils\util.py", line 112, in convert_path
        raise ValueError("path '%s' cannot end with '/'" % pathname)
    ValueError: path 'yappi/' cannot end with '/'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\dimam\AppData\Local\Temp\pip-install-b6ze0_h7\yappi\

```

**Comment by sumerc** [comment]
Ahhh... Windows again. 

I will fix this ASAP and integrate one CI build for Windows. Could you confirm Yappi works with that line of change you suggested?

**Comment by klaipher** [comment]
> Ahhh... Windows again.
> 
> I will fix this ASAP and integrate one CI build for Windows. Could you confirm Yappi works with that line of change you suggested?

yes, everything works. I already used yappi in PyCharm.

**Comment by sumerc** [comment]
- This and couple of compile errors on different Python versions on Windows is fixed.
- Integrated Appveyor for Windows tests.

I will hopefully release a new version tomorrow.

**Comment by ruicao93** [comment]
> * This and couple of compile errors on different Python versions on Windows is fixed.
> * Integrated Appveyor for Windows tests.
> 
> I will hopefully release a new version tomorrow.

Hi Sümer, 

Just a kink reminder that the issue have widely influenced the installation of libs which depend on the pypi, e.g. future

Could you help update the release version ASAP? Many thanks for your contribution.

Thanks,
Rui

**Comment by sumerc** [comment]
Hi Rui.

I have released a new version today: 1.2.3 in PyPI(+ Windows CI support). Please could you confirm there is no issue on your end.

**Comment by ruicao93** [comment]
> Hi @ruicao93,
> 
> I have released a new version today: 1.2.3 in PyPI(+ Windows CI support). Please could you confirm there is no issue on your end.

Hi @sumerc ,

I verified the installation on the windows today, yappi can be installed successfully now.

Thanks a lot, Sümer. I think the issue could be closed.
